webpack.config.prod.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 						loader: 'css-loader',
 						options: {
 							modules: {
-								localIdentName: '[folder]-[local]-[hash:base64:5]',
+								localIdentName: '[folder]-[local]-[contenthash:base64:5]',
```

Comment:
[note] Webpack now encourages `[contenthash]` instead of `[hash]`.
See also https://webpack.js.org/guides/caching/#output-filenames